src/common/Types.ts

```diff
@@ -488,7 +488,7 @@ export interface ImagePropsShared extends CommonProps {
     resizeMethod?: 'auto' | 'resize' | 'scale'; // Android only
     title?: string;
 
-    onLoad?: (size: Dimensions) => void;
+    onLoad?: (size: OptionalDimensions) => void;
```

Comment:
Why are these fields optional? Are there circumstances where RN can't hand back valid dimensions? I'd prefer if these are non-optional. Otherwise it complicates the logic in all components that use this callback.